examples/volumes/empty_dir/README.md

```diff
@@ -0,0 +1,74 @@
+# EmptyDir volume plugin with HugePages medium
```

Comment:
we can provide an example in a follow-on, split this into a separate pr that is a follow-on.